src/reporter.ts

```diff
@@ -115,22 +114,61 @@ export function scrubSpans(resultSpans: any[]) {
   return resultSpans.filter((span) => safeExecute(scrubSpan, 'Failed to scrub span')(span));
 }
 
+export function getPrioritizedSpans(spans: any[], maxSendBytes: number): any[] {
+  logger.debug('Using smart spans prioritization');
+  spans.sort(spansPrioritySorter);
+  let currentSize = 0;
+  const spansToSendSizes = {};
+  const spansToSend = {};
+
+  // First we try to take only the spans metadata
+  for (let index = 0; index < spans.length; index++) {
+    const spanMetadata = getSpanMetadata(spans[index]);
+    spansToSendSizes[index] = 0;
+    if (spanMetadata == undefined) continue;
```

Comment:
Not much of a difference in this case, but `===` is the de-facto standard:
```suggestion
    if (spanMetadata === undefined) continue;
```